frontend/layout/lib/layoutModel.ts

```
@@ -472,6 +486,79 @@
+                            }
+                            const newNode = newLayoutNode(undefined, action.nodesize, undefined, {
+                                blockId: action.blockid,
+                            });
+                            const splitAction: LayoutTreeSplitHorizontalAction = {
+                                type: LayoutTreeActionType.SplitHorizontal,
+                                targetNodeId: targetNode.id,
+                                newNode: newNode,
+                                position: action.position,
+                            };
+                            this.treeReducer(splitAction, false);
+                        }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)